Update Cloud/Portal articles (#2410)
umbraco/UmbracoDocs: Umbraco-Cloud/Set-Up/Manage-Environments/index.md

# Manage environments

When working with an Umbraco Cloud project you can add and remove extra environments. 

> For the starter plan you can add a development environment for an additional price.

Reviewer: ```suggestion
For the Starter plan you can add a Development environment for an additional price per month.
```

> for the standard plan you get the development environment for free and can add a staging environment for an additional price.

Reviewer: ```suggestion
For the Standard plan you get the Development environment for free and can add a Staging environment for an additional price per month.
```

>  [You can find information about the prices for the environments here](https://umbraco.com/cloud-pricing/)

Reviewer: ```suggestion
 [Learn more about the additional prices on Umbraco Cloud](https://umbraco.com/cloud-pricing/)
```
